components/SliderPlusDescription.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <v-carousel>
+    <v-carousel-item
+      v-for="(slide, index) in slides"
+      :key="index"
+      :src="item.src"
+      cover
```

Comment:
@AgnieszkaIwan according to a clickup's task there should be passed also an alt attribute.